docs/observability.md

```
@@ -0,0 +1,255 @@
+
+```json
+{
+  "Otel": {
+    "OtlpEndpoint": "http://otel-collector:4317"
+  }
+}
+```
+
+**Option B — environment variable:**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)